nomos-da/network/core/src/protocols/replication/behaviour.rs

```diff
@@ -80,10 +80,7 @@ impl Clone for ReplicationError {
                     OpenStreamError::Io(error) => {
                         OpenStreamError::Io(std::io::Error::new(error.kind(), error.to_string()))
                     }
-                    err => OpenStreamError::Io(std::io::Error::new(
-                        std::io::ErrorKind::Other,
-                        err.to_string(),
-                    )),
+                    err => OpenStreamError::Io(std::io::Error::other(err.to_string())),
```

Comment:
Why does the formatting keep changing? 😅

Reply:
No it's not just a formatting change. We changed how we map the inner error. We replaced `std::io::Error::new(std::io::ErrorKind::Other, err.to_string())` with `std::io::Error::other(err.to_string())`